unicodetools/src/main/java/org/unicode/text/UCA/UCA.java

```diff
@@ -207,12 +205,19 @@ public Iterator<Primary> iterator() {
     }
 
     /**
-     * Initializes the collation from a stream of rules in the normal formal. If the source is null,
-     * uses the normal Unicode data files, which need to be in BASE_DIR.
-     *
-     * @param type
+     * Initializes the collation from a stream of rules in the allkeys.txt format. If the source is
+     * null, uses the normal Unicode data files, which need to be in BASE_DIR.
+     */
+    public UCA(String sourceFile, String unicodeVersion) throws java.io.IOException {
```

Comment:
FYI: Same for the UCA constructor. The implementation function now takes two primaries instead of the obsolete class Remap, but I added a convenience constructor for the DUCET, without additional parameters that would have to be "nulled".